tests/utils.js

```diff
@@ -327,10 +496,19 @@ const getDefaultSettings = () => {
   return JSON.parse(fs.readFileSync(pathToDefaultAppSettings).toString());
 };
 
+function deprecated(name, replacement) {
```

Comment:
Good work. I managed to run a couple of tests and the deprecated warnings show. Not sure if we should stick to arrow function for newly written code?

Reply:
That's my bad. I'll update to be an arrow function